Makefile

```diff
@@ -36,10 +35,7 @@ docker-compile:
 # For cases where we're building from local
 docker-build:
 	docker build --rm -t ${IMAGE} rootfs
-
-# Push to a registry that Kubernetes can access.
-docker-push:
-	docker push ${IMAGE}
+	docker tag -f ${IMAGE} ${MUTABLE_IMAGE}
```

Comment:
Nah, let's just leave it for now. It doesn't cause problems as-is, but I can't be certain that removing it won't cause any.